homeassistant/components/weatherflow/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "weatherflow",
+  "name": "WeatherFlow",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/weatherflow",
+  "requirements": ["pyweatherflowudp==1.4.1"],
+  "ssdp": [],
+  "zeroconf": [],
+  "homekit": {},
+  "dependencies": [],
+  "codeowners": ["@natekspencer", "@jeeftor"],
+  "iot_class": "local_polling"
```

Comment:
`iot_class` should be `local_push`
Add `"integration_type": "hub", "loggers": ["pyweatherflowudp"]` here as well